packages/react-button/package.json

```diff
@@ -17,17 +17,19 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "start": "yarn storybook",
+    "test": "jest",
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/packages/react-button/src && yarn docs",
+    "storybook": "start-storybook"
   },
   "devDependencies": {
     "@fluentui/a11y-testing": "^0.1.0",
     "@fluentui/babel-make-styles": "^9.0.0-alpha.25",
     "@fluentui/eslint-plugin": "^1.3.2",
     "@fluentui/jest-serializer-make-styles": "^9.0.0-alpha.25",
     "@fluentui/react-conformance": "^0.4.3",
+    "@fluentui/react-menu": "^9.0.0-alpha.50",
```

Comment:
can we get rid of this ?

- we don't specify devDeps for vNext packages that are used solely in story files
- you added it probably to make CI pass right ? ([issue we already run into](https://github.com/microsoft/fluentui/issues/18752))
   - let's go with "standard" if that's the case 
![2021-07-16 at 14 38](https://user-images.githubusercontent.com/1223799/125948918-47d5f93d-2575-42e2-96cd-c1782d544fa7.png)

Reply:
Just removed it! And thanks for the clarification 😄

Reply:
don't forget to add that `@ts-ignore` 

CI is failing because of it (as expected) 
![2021-07-19 at 15 26](https://user-images.githubusercontent.com/1223799/126167113-a1b18a10-209f-48aa-b0a5-9c636784e2c6.png)